### PR TITLE
Display TPV data to merchants

### LINKED ISSUE #8593: Add TPV data to TotalPaymentsVolume Component
### Description
Add TPV data to TotalPaymentsVolume Component

### PR DESCRIPTION
**Pull request by naman03malhotra**
Fixes #8593 

#### Changes proposed in this Pull Request
Added TPV data to TotalPaymentsVolume Component

#### Testing instructions

- Refresh your cache
image
- Verify if the data is loading correctly

-------------------

- [ ] Run `npm run changelog` to add a changelog file, choose `patch` to leave it empty if the change is not significant. You can add multiple changelog files in one PR by running this command a few times. 
- [ ] Covered with tests (or have a good reason not to test in description ☝️)
- [ ] Tested on mobile (or does not apply)

**Post merge**

- [ ] Link to testing instructions from [release testing doc](https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions) following [these instructions](https://github.com/Automattic/woocommerce-payments/wiki/How-to-write-good-manual-testing-scenarios) : _Add link here / 'QA Testing Not Applicable'_
- [ ] Add or update [critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Critical-flows) and [testing instructions for critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Testing-instructions-for-critical-flows), if applicable.
- [ ] Add what's changed (description, screenshot, demo videos etc.) to the release announcement post, if applicable.

### CODE REVIEW

**Comment by naman03malhotra** [comment]
@Automattic/helix does the TPV data will also be filtered based on dates, or will it always be the Total Payments Volume for that merchant?

**Comment by botwoo** [comment]
### Test the build 

#### Option 1. Jetpack Beta

- Install and activate [Jetpack Beta](https://jetpack.com/download-jetpack-beta/).
- Use this build by searching for PR number `8592` or branch name `feat/8519-add-tpv-data` in your-test.site/wp-admin/admin.php?page=jetpack-beta&plugin=woocommerce-payments          

#### Option 2. Jurassic Ninja - available for logged-in A12s

:rocket: [Launch a JN site with this branch](https://jurassic.ninja/create/?jetpack-beta&shortlived&nojetpack&woocommerce&woocommerce-payments-dev-tools&branches.woocommerce-payments=feat/8519-add-tpv-data) :rocket:

:information_source: Install this [Tampermonkey script](https://github.com/Automattic/woocommerce-payments/tree/develop/bin/wcpay-live-branches) to get more options.

---

Build info: 

- Latest commit: 3ad22d85979a2bdd2f7d497b6be007664ffc0259
- Build time: 2024-04-09 12:24:36 UTC

_Note: the build is updated when a new commit is pushed to this PR._

**Review by jessy-p — COMMENTED** [review]
@naman03malhotra  LifetimeTPV from settings is the payments volume for entire lifetime of merchant in USD. It was introduced for showing empty state for merchants who have no transactions at all - https://github.com/Automattic/woocommerce-payments/issues/8493

This Payments Volume should be in the currency selected (for now, store currency) and should be changing according to the filters selection. So we may use the new API introduced in https://github.com/Automattic/woocommerce-payments-server/pull/5007

You can change and base this PR off of `add/8519-reporting-api-to-component` branch introduced in this PR - https://github.com/Automattic/woocommerce-payments/pull/8589

**Comment by nagpai** [comment]
@naman03malhotra - I have a PR just presented for review,  where the same component can be reused for TPV and the 4 tiles.
https://github.com/Automattic/woocommerce-payments/pull/8564

This would remove the `TotalPaymentsVolume` component and use a `PaymentDataTile` component.

**Comment by naman03malhotra** [comment]
>This would remove the TotalPaymentsVolume component and use a PaymentDataTile component.

@nagpai I'll have a look at your PR, but that said, it makes sense for your PR to go first and then we'll intergrate data back into it.

File: client/components/payments-activity/test/__snapshots__/index.tsx.snap [hunk 1]
```diff
@@ -49,7 +49,7 @@ exports[`PaymentsActivity component should render 1`] = `
             <div
               class="total-payments-volume__body"
             >
-              $30.00
+              $10.00
```
Comment (jessy-p) [inline client/components/payments-activity/test/__snapshots__/index.tsx.snap:52]: Maybe default value can be $0.00 ?

**Pull request merged**